src/sgame/sg_spawn_shared.cpp

```diff
@@ -69,6 +69,14 @@ void think_aimAtTarget( gentity_t *self )
 	VectorAdd( self->r.absmin, self->r.absmax, origin );
 	VectorScale( origin, 0.5f, origin );
 
+	if ( false
+			|| self->s.origin2[0] != 0.f
```

Comment:
Do you need to compare with epsilon?

Reply:
No, because the values are set to 0 before. If it happens that something changed the value and all result in a 0 +- epsilon, then it means something was initialized and thus it's fine (but map is likely buggy).